slurm-on-gke/README.md

```diff
@@ -0,0 +1,448 @@
6X+Z0pJCaHt0atHfb7233pqwWTllxSqe5MuOrnR0ercsJ1Xy2HfCCCAAAIIIIAAAgi4VmDZ9lX6ZutPmp+xVCUq1W3HXKPLj/yNa/fLxhBAAAEEoivgusDOKXf9dum+v0grtoRe/NknSQ9fFPrx5Ufe+Jq0fHVo58WVSr/uIj3429CO98NRK7PW6qOVX+j91Z8pI2+HOtdvo+5NuqpTels1SW2o5PgkP5RJDQgggAACCCCAAAIIGBXILy5Uxn8fQ7Nsx2rNz1iiZVlr1CS5gXq37anftDtTHdLbGN0PiyGAAAIIuEvAlYGd8wy5fy+X7n1TKg7R64hW0quDyw7eLqlhDeft+/MLnpRy80JbpFsb6cErpYaHhXa8n47aWZCtT9bM10erP9d3W3/aW1rzlMPVLLWBWqQ2Vt3Eqj6c7CcFakEAAQQQQAABBBBAoPYC2YW52rR7W/B50et2b917oV81OloXtD1b57TprnpJFv7HRu1JORMBBBDwrYArAztH23n23NzvpUf+Fpr99AeklpJunSotXyc1SpcOS5GSkqRAnFRSIuXnS1nZUmaOdPNV0lUdpak/SZPfPvgaqUnSc9dKR7X073PrDq5QdsTCzUu1IWezNmRnaF3OZq3L3qjV2ZuUVRDiK35DXYjjEEAAAQQQQAABBBDwkUB6Uj21r9tCbeq2UMvDmqp5amM1TW2kE5p28VGVlIIAAgggEAkB1wZ25cVNmCO9/XXNb47t00sa0UPaKGngGKmw6OA0zRpJfxladtwNk6UV66s/Jy1JGn2JdHbXg1+XIxBAwG6B0dNzgwBP9E+xG4LqEUAAAQQQQAABBBBAAAEEai3g+sBuR7b06LvSN6uqrjE5UZp9T9nPJi+SpoZ4R55zfPkbZid+J03/oOrrJ8ZJt/SSrjhd8ukLYWs9PJyIAAKVBQjsmAoEEEAAAQQQQAABBBBAAIFwBVwf2DkFbtkpXT9Byq7mWXPX95UuOk4a9JiUVxg6Sd006YW7pD9OkZavrfq8c46W7u4rpSSGfl2ORAABewUI7OztPZUjgAACCCCAAAIIIIAAApES8ERg5xS7eLX08HtSRlakSj/4dbq0kB76rdS0/sGP5QgEEEDAESCwYw4QQAABBBBAAAEEEEAAAQTCFfBMYOe8NOL9hdK42VJBCM+oCxemRQPp2YFSs/Rwr8T5CCBgkwCBnU3dplYEEEAAAQQQQAABBBBAIDoCngnsnPILiqXnP5TeXySVlkYHxLlqvWTp7oulM7rwRtjoKXNlBPwpQGDnz75SFQIIIIAAAggggAACCCBgUsBTgZ0Dk5MrPfm+9NlP0WO6uad0VQ8pMT56a3BlBBDwpwCBnT/7SlUIIIAAAggggAACCCCAgEkBzwV2Ds7mTOmBt6WfN0X2Tru4gHThsdLoPiZbwFoIIOAnAQI7P3WTWhBAAAEEEEAAAQQQQACB2AhEPbBbsrZYw6bl6feX1tE5xyRErMqsXGnV5sg9zy5OUnqa1K6JlMCddRHrExdCwDaBy57ZrYuPT9CNvZJsK516EUAAAQQQQAABBBBAAAEEIiQQ9cAut6BUvZ/erRvPTtI1PRIjtG0ugwACCLhPYN32El03IVf3XFJH5x4bub+gcF+l7AgBBBBAAAEEEEAAAQQQQCCaAlEP7JzNOx8Ra1o/oBEX14lmLVwbAQQQiKnAFz8V6Q8z8vXCdcnq3JJbdWPaDBZHAAEEEEAAAQQQQAABBDwsYCSwm/Jpgb5eUawx/ZLVIM358ClfCCCAgP8EnpqVr48WF+m9O1NVN4Xf6/zXYSpCAAEEEEAAAQQQQAABBMwIGAnsNmwv0bCpeTqmVUAPXZlspjJWQQABBAwKTP28QK99WaiBpyfq+p48v84gPUshgAACCCCAAAIIIIAAAr4TMBLYOWoz/l2o8XMLeJad70aIghBAYMXmYt00OU/tGwX0ys0pgCCAAAIIIIAAAggggAACCCAQloCxwM7Z5V3TcrVobYkeujxZZ3bm+U5hdY6TEUDAFQIZWSW65oXc4F5eGZKi9k0DrtgXm0AAAQQQQAABBBBAAAEEEPCugNHAzmHqPz5Xm3eWqO+JCRp0VpLqp/KcJ++ODztHwG6BT34s0qN/yw8i8CZsu2eB6hFAAAEEEEAAAQQQQACBSAoYD+yczc/4ulDjPy5Q6wYB9flVgto1DujIZgEdxkPaI9lbroUAAlEQWL2lRCs2l+j7NcWatahIJ7ULaGTvOmpcjzvrosDNJRFAAAEEEEAAAQQQQAABKwViEtg50t+tKNYLcwq0elvJXvi2hwfUuK6VfaBoBBDwgMCyjaXKzi/du9PbeyXpslMTPbBztogAAggggAACCCCAAAIIIOAlgZgFdg5SflGp1m8r0brtpVq7rURrtpYoc3fFfwx7CZK9IoCA/wXaN45X20YBtWkUp9aNAqrHXcH+bzoVIoAAAggggAACCCCAAAIxEIhpYBeDelkSAQQQQAABBBBAAAEEEEAAAQQQQAABVwsQ2Lm6PWwOAQQQQAABBBBAAAEEEEAAAQQQQMA2AQI72zpOvQgggAACCCCAAAIIIIAAAggggAACrhYgsHN1e9gcAggggAACCCCAAAIIIIAAAggggIBtAgR2tnWcehFAAAEEEEAAAQQQQAABBBBAAAEEXC1AYOfq9rA5BBBAAAEEEEAAAQQQQAABBBBAAAHbBAjsbOs49SKAAAIIIIAAAggggAACCCCAAAIIuFqAwM7V7WFzCCCAAAIIIIAAAggggAACCCCAAAK2CRDY2dZx6kUAAQQQQAABBBBAAAEEEEAAAQQQcLUAgZ2r28PmEEAAAQQQQAABBBBAAAEEEEAAAQRsEyCws63j1IsAAggggAACCCCAAAIIIIAAAggg4GoBAjtXt4fNIYAAAggggAACCCCAAAIIIIAAAgjYJkBgZ1vHqRcBBBBAAAEEEEAAAQQQQAABBBBAwNUC/x9KAxtB1qcYDgAAAABJRU5ErkJggg==>
```

Comment:
This is an unusual way to embed images. Usually we store the image in a separate file and reference the path in the README

Reply:
agree, changed.